config_builder.go

```diff
@@ -687,7 +687,7 @@ func (d *RPCSignerWalletImpl) BuildChainControl(
 
 	rpcKeyRing, err := rpcwallet.NewRPCKeyRing(
 		baseKeyRing, walletController,
-		d.DefaultWalletImpl.cfg.RemoteSigner,
+		d.DefaultWalletImpl.cfg.RemoteSigner, walletConfig.CoinType,
```

Comment:
Great commit message here, really clear how everything fits together now! Given we can just fully specify everything, the signer itself can really just be stateless other than the initial HD key info 😎